### ISSUE TITLE
⚠️  WARN Error parsing undefined: SyntaxError: Unexpected token u 

### ISSUE THREAD

**Issue by cdiaz** [Floobits/floomatic#27]
floomatic version 0.5.1
node v4.0.0
Ubuntu 14.10

**Comment by ggreer** [comment]
Could you please give more info on what broke and how? Is there a stack trace? How did you run floomatic? What OS are you using?

Basically, without steps to reproduce the error, I can't fix it.

**Comment by seeya** [comment]
I'm getting the same error while trying to do 
`floomatic --read-only --join https://floobits.com/owner_name/workspace_name`
Running on a Linux, Ubuntu. Node v6.3.0.
```
   WARN 2016-07-12 04:10:33 EDT Error parsing undefined: SyntaxError: Unexpected token u in JSON at position 0
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received false
    at assertPath (path.js:7:11)
    at basename (path.js:1357:5)
    at /usr/local/lib/node_modules/floomatic/node_modules/lodash/lodash.js:4631:46
    at Object.exports.run (/usr/local/lib/node_modules/floomatic/floomatic.js:99:54)
    at Object.<anonymous> (/usr/local/lib/node_modules/floomatic/bin/floomatic:3:25)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

**Comment by ggreer** [comment]
This should be fixed in the latest version (v0.5.4).

**Comment by alexozer** [comment]
This now seems to occur on Arch Linux with the latest version of floomatic:

```
$ node -v
v7.7.1

$ npm list -g floomatic
/usr/lib
└── floomatic@0.5.5 

$ floomatic
⚠️   WARN 2017-03-03 21:08:21 EST Error parsing undefined: SyntaxError: Unexpected token u in JSON at position 0
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received false
    at assertPath (path.js:7:11)
    at basename (path.js:1355:5)
    at /usr/lib/node_modules/floomatic/node_modules/lodash/lodash.js:5154:46
    at Object.exports.run (/usr/lib/node_modules/floomatic/floomatic.js:121:56)
    at Object.<anonymous> (/usr/lib/node_modules/floomatic/bin/floomatic:3:25)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```
